samza-core/src/main/java/org/apache/samza/zk/ZkCoordinationServiceFactory.java

```diff
@@ -26,12 +26,10 @@
 
 
 public class ZkCoordinationServiceFactory implements CoordinationServiceFactory {
-
-
   synchronized public CoordinationUtils getCoordinationService(String groupId, String participantId, Config config) {
     ZkConfig zkConfig = new ZkConfig(config);
     ZkClient zkClient = new ZkClient(zkConfig.getZkConnect(), zkConfig.getZkSessionTimeoutMs(), zkConfig.getZkConnectionTimeoutMs());
-    ZkUtils zkUtils = new ZkUtils(participantId, new ZkKeyBuilder(groupId), zkClient, zkConfig.getZkConnectionTimeoutMs());
```

Comment:
@nickpan47 @navina 
I remember I preferred (and still prefer) calling this `participantId` instead of `processorId` when I reviewed Boris's PR.  

It keeps the scope of this interface self-contained as a general purpose coordination API which is independent of `StreamProcessor` (or even `Samza` for that matter).